enterprise/admin/__init__.py

```diff
@@ -75,6 +90,14 @@ def logo(instance):
             return instance.branding_configuration.logo
         return None
 
+    @staticmethod
+    def identity_provider(instance):
+        """
+        Instance is EnterpriseCustomer.
+        """
+        ecip = instance.enterprise_customer_identity_provider
```

Comment:
nit: can we keep it like `ec_identity_provider / ec_idp` or just `identity_provider` ?